src/command_modules/azure-cli-network/azure/cli/command_modules/network/custom.py

```diff
@@ -966,7 +966,11 @@ def export_zone(cmd, resource_group_name, zone_name):
 
             zone_obj[record_set_name][record_type].append(record_obj)
 
-    print(make_zone_file(zone_obj))
+    zone_file = make_zone_file(zone_obj)
```

Comment:
A better variable name is `zone_file_content` to distinguish it from an actual file object, which is created in following code.